spec/fixtures/application/1.0/application.json

```diff
@@ -202,7 +202,8 @@
             "country": "country_x",
             "postcode": "postcode_x" }
         }
-      ]
+      ],
+      "has_frozen_income_or_assets": null
```

Comment:
Set as `null` as `has_frozen_income_or_assets` has been set in the income details section